### DISCUSSION TITLE
Stream Deck + Enhancements

### DISCUSSION THREAD

**Discussion by jer78** [cgiesche/streamdeck-homeassistant#203] [Feature Requests]
First off, thank you very much for Stream Deck + support! This is amazing. 

Just inquiring on a few enhancements. 

1) Option for transparent backgrounds (saw this is being worked on already)

2) The option to include a visual bar for values with templating. For example, if I use it to adjust volume, a scale of 1-100% and reflects the current value. This would probably need templating since media_players use a float rather than a percentage. 

3) Use same fonts by default as the built in Stream Deck dials.  

4) A custom icon that is the same size and location as the built in dials. 

Here's a screenshot of the current HA dial vs the built in dial. 

![image](https://github.com/cgiesche/streamdeck-homeassistant/assets/1253286/eadf49c5-6483-4a43-9e60-b35424eac4ff)

**Comment by cgiesche** [comment]
Hi,

1. yep, im working on this (#222)
2. mmhm, mhmm, i have to think about this. It is possible (as seen on the left) but may lead to a more complex configuration
3. if you know details (font, font size) let me know. I will definetly work on the appearance in general.
4. maybe not custom icons, but the icon provided by home-assistant could be possible.
